src/styles/index.scss

```diff
@@ -1,3 +1,17 @@
+@import './utils/variables';
+@import 'main';
+
+* {
```

Comment:
Hello, @renata-aline.

According to Code Style Rules, it is not recommended to use the * selector because it can impact the performance.

Check if it can be changed.